README.md

```diff
@@ -1,24 +1,55 @@
 # OpenCompose
 
-[![Build Status](https://travis-ci.org/redhat-developer/opencompose.svg?branch=master)](https://travis-ci.org/redhat-developer/opencompose)
+[![Build Status Widget]][Build Status] [![GoDoc Widget]][GoDoc] [![Slack Widget]][Slack]
 
-The goal of OpenCompose is to make it easier for developers to on-board to Kubernetes.
 OpenCompose is a declarative higher level abstraction for specific Kubernetes resources.
 
-Very simple idea, isn't it? 
 A developer shouldn't have to learn various Kubernetes concepts just to test and deploy their applications.
 Focus on the application that is being developed.
 
+The goal of OpenCompose is to make it easier for developers to on-board to Kubernetes.
+
 We are at a very evolving stage of this project and we have listed some of our ideas as [issues](https://github.com/redhat-developer/opencompose/issues)
 and [examples](https://github.com/redhat-developer/opencompose/blob/master/examples/).
 We are open to suggestions and contributions from the Kubernetes community as our project grows.
 Please send any PRs, issues or RFCs to improve this project.
 
+## Use Case
+
+Go from a simple [hello-nginx.yaml](https://github.com/redhat-developer/opencompose/blob/master/examples/hello-nginx.yaml) example to a full Kubernetes environment:
+
+Create (or download) `hello-nginx.yaml`
+
+```yaml
+version: 0.1-dev
+services:
+- name: helloworld
+  containers:
+  - image: nginx
+    ports:
+    - port: 8080:80
```

Comment:
yeh, It can be bit confusing :-( if you come from docker/docker-compose. OpenCompose has a different order for ports. But there is a good reason for it ;-)